README.md

```diff
@@ -1,8 +1,10 @@
 # Optimizely VSCode Extension
 
-The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript typescript and react.  If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
+The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript, typescript and react. If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
```

Comment:
The Optimizely VSCode extension autocompletes:
- feature keys
- experiment keys
- feature variables 
in Javascript, Typescript, and React.

QUESTION: it's not clear if In .js, .ts, and react, you can also autocomplete attribute and event keys without keyboard shortcuts? if you can, please add them to the list and modify the second sentence:
In other languages, you can insert the preceding list of elements using the following keyboard shortcuts.

Reply:
I believe we can, so they should be added to the list. @thomaszurkan-optimizely WDYT?

Reply:
events and attributes don't auto-complete.

Reply:
I added back events tags since we added it in our last #14 pr